cmd/protoc-gen-go-grpc/grpc_test.go

```diff
@@ -0,0 +1,97 @@
+/*
+ *
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"reflect"
+	"strings"
+	"testing"
+)
+
+func TestProtocBinary(t *testing.T) {
+	workingDir, err := os.Getwd()
```

Comment:
I don't think we need this `workingDir` at all.  `os.MkdirTemp` uses the default temp directory if the first parameter is `""` which seems perfectly fine and `command.Dir` defaults to the current directory already.

Reply:
make sense, will do